mypy_ls/plugin.py

```diff
@@ -168,25 +168,25 @@ def pyls_lint(config: Config, workspace: Workspace, document: Document,
             "silent"
         ])
 
-        log.info(f"executing mypy args={args}")
+        log.info(f"executing mypy args = %s", args)
```

Comment:
This is an "empty" f-string, i.e. it doesn't have any `{}` formatting blocks. It's harmless but flake8 usually marks it as an error.